airlock/templates/workspaces.html

```diff
@@ -5,19 +5,30 @@
 {% block content %}
   <div class="flex flex-col gap-4 max-w-3xl">
     {% airlock_header title="Workspaces for "|add:request.user.username %}
-
     {% for project, workspaces in projects.items %}
-      {% #card title=project.display_name %}
-        {% #list_group id="workspaces" %}
-          {% for workspace in workspaces %}
-            {% #list_group_item href=workspace.get_url %}{{ workspace.display_name }}{% /list_group_item %}
-          {% endfor %}
-        {% /list_group %}
-      {% /card %}
-    {% empty %}
-      {% #card title="Workspaces" container=True %}
-        {% list_group_empty title="No workspaces available" description="You do not have access to any workspaces" %}
-      {% /card %}
-    {% endfor %}
-  </div>
-{% endblock content %}
+      {% #card %}
+        <details class="group" open>
+          <summary class="list-none more-dropdown__btn">
+            {% fragment as custom_button %}<span class="opacity-75 ml-1"><img height="16" width="16" class="icon inline-block group-open:hidden" src="/static/icons/keyboard_arrow_down.svg" alt="">
```

Comment:
This is pedantic (sorry), but I think it would be better to increase the size of the up/down arrows so they're more in line with the size of the project name text.  Currently they're the same size as they are in the nav bar dropdowns, and I almost missed them altogether.
height/width =30 is about right, I think (you'll need to remove the `icon` class because it overrides the height/width)

![image](https://github.com/user-attachments/assets/efc9d8d4-b55e-46ac-8684-ebf2d2b65dba)